ops/demo-deploy.tmpl.yaml

```diff
@@ -59,27 +59,27 @@ extraEnvVars: &envVars
   - name: DB_ADAPTER
     value: postgresql
   - name: DB_HOST
-    value: staging-postgresql
+    value: postgres-cluster-alpha-ha.postgres.svc.cluster.local
   - name: DB_NAME
-    value: hyku
+    value: pals-demo-hyku
   - name: DB_PASSWORD
     value: $DB_PASSWORD
-  - name: DATABASE_URL
-    value: postgresql://postgres:$DB_PASSWORD@pg-postgresql.staging-postgres.svc.cluster.local/staging-palni-palci?pool=5
+  - name: DB_URL
```

Comment:
Saw your comment on utk-hyku prod deploy, so does this need to be defined here?

Reply:
it does not